Compos/Helpers/UnitsHelper.cs

```diff
@@ -29,191 +39,13 @@ public static DensityUnit DensityUnit {
-
-              default:
-                return PressureUnit.Undefined;
-            }
-          default:
-            return PressureUnit.Undefined;
-        }
-      }
-    }
-    public static ForcePerLengthUnit ForcePerLengthUnit {
-      get {
-        switch (LengthUnitGeometry) {
-          case LengthUnit.Millimeter:
-            switch (ForceUnit) {
-              case ForceUnit.Newton:
-                return ForcePerLengthUnit.NewtonPerMillimeter;
-
-              case ForceUnit.Kilonewton:
-                return ForcePerLengthUnit.KilonewtonPerMillimeter;
-
-              case ForceUnit.Meganewton:
-                return ForcePerLengthUnit.MeganewtonPerMillimeter;
-
-              case ForceUnit.TonneForce:
-                return ForcePerLengthUnit.TonneForcePerMillimeter;
-
-              default:
-                return ForcePerLengthUnit.Undefined;
-            }
-          case LengthUnit.Centimeter:
-            switch (ForceUnit) {
-              case ForceUnit.Newton:
-                return ForcePerLengthUnit.NewtonPerCentimeter;
-
-              case ForceUnit.Kilonewton:
-                return ForcePerLengthUnit.KilonewtonPerCentimeter;
-
-              case ForceUnit.Meganewton:
-                return ForcePerLengthUnit.MeganewtonPerCentimeter;
-
-              case ForceUnit.TonneForce:
-                return ForcePerLengthUnit.TonneForcePerCentimeter;
-
-              default:
-                return ForcePerLengthUnit.Undefined;
-            }
-          case LengthUnit.Meter:
-            switch (ForceUnit) {
-              case ForceUnit.Newton:
-                return ForcePerLengthUnit.NewtonPerMeter;
-
-              case ForceUnit.Kilonewton:
-                return ForcePerLengthUnit.KilonewtonPerMeter;
-
-              case ForceUnit.Meganewton:
-                return ForcePerLengthUnit.MeganewtonPerMeter;
-
-              case ForceUnit.TonneForce:
-                return ForcePerLengthUnit.TonneForcePerMeter;
-
-              default:
-                return ForcePerLengthUnit.Undefined;
-            }
-          case LengthUnit.Foot:
-            switch (ForceUnit) {
-              case ForceUnit.PoundForce:
-                return ForcePerLengthUnit.PoundForcePerFoot;
-
-              case ForceUnit.KilopoundForce:
-                return ForcePerLengthUnit.KilopoundForcePerFoot;
-
-              default:
-                return ForcePerLengthUnit.Undefined;
-            }
-          case LengthUnit.Inch:
-            switch (ForceUnit) {
-              case ForceUnit.PoundForce:
-                return ForcePerLengthUnit.PoundForcePerInch;
-
-              case ForceUnit.KilopoundForce:
-                return ForcePerLengthUnit.KilopoundForcePerInch;
-
-              default:
-                return ForcePerLengthUnit.Undefined;
-            }
-          default:
-            return ForcePerLengthUnit.Undefined;
-        }
-      }
-    }
```

Comment:
is this dead code? That is why we removed it, or you replaced it with something else?

Reply:
@psarras not used anywhere and hence removed